docs/UserGuide.md

```diff
@@ -1,186 +1,239 @@
+> There will be 3 tabs, which can be toggled using this command.
 
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+### 2.6. Deleting a task: `delete`
+Deletes the specified task.
+Format: `delete INDEX_NUMBER`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> Task labeled by INDEX_NUMBER will be deleted from the list.<br>
+> INDEX_NUMBER of task is shown as according to the current tab.
 
 Examples:
+* `delete 2`
+* `delete 5`
 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+### 2.7. Completion of a task: `complete`
 
-### 2.6. Deleting a person : `delete`
+Marks the specified task as complete.<br>
+Format: `complete INDEX_NUMBER`
+> The task at INDEX_NUMBER will be marked as completed.<br>
+> INDEX_NUMBER of task is shown as according to the current tab.
 
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Examples:
+* `complete 2`
+* `complete 5`
 
-> Deletes the person at the specified `INDEX`. <br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+### 2.8. Track overall progress: `progress`
 
-Examples:
+Shows all the tasks completed over time and also specifying the tasks overdued.<br>
+> A general overview of performance/efficiency based on the usage of the task manager will be shown.<br>
 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+Format: `progress NUMBER_OF_DAYS`
+> NUMBER_OF_DAYS is a interger which indicates the your progress for these past days (excluding today).<br>
+> It will provide an analysis on the number of task that you have completed/overdue.
 
-### 2.7. Select a person : `select`
+Examples:
+* `progress 7`
 
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+### 2.9. Search for tasks: `search`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+Searches for tasks based on keywords by user's input.<br>
 
-Examples:
+Format: `search KEYWORD`
+> A valid input needs to be keyed in. No special characters such as ASCII whitespace is allowed.<br>
+> A popout will appear to show you the list of tasks that contains the stated `KEYWORD`
```

Comment:
You seem to like to use "popouts".  How do you close them?  Consider how Jim would like to deal with them.